include/lsst/afw/geom/SphPoint.h

```diff
@@ -0,0 +1,319 @@
+// -*- LSST-C++ -*-
+
+/*
+ * LSST Data Management System
+ * See COPYRIGHT file at the top of the source tree.
```

Comment:
The confluence page seems to say the same thing:
> * The LSST Copyright statement appropriate for the file type. Refer to:
>     - git@git.lsstcorp.org:LSST/DMS/devenv/templates.git  {CopyrightHeader.py, CopyrightHeader.h, CopyrightHeader.cc}

I can't clone that URI (is it out of date?), but I assume it's the same code found at https://dev.lsstcorp.org/trac/browser/DMS/devenv/templates/trunk. See also https://confluence.lsstcorp.org/display/LDMDG/Documentation+Standards#DocumentationStandards-Example.hfile.